src/types/users/students.ts

```diff
@@ -0,0 +1,42 @@
+import {ExpectedContractType, ExpectedWorkType} from "../../entities/students-entity";
+
+export interface AvailableStudentData {
+  id: string;
+  fullName: string;
+  courseCompletion: string;
+  courseEngagement: string;
+  projectDegree: string;
+  teamProjectDegree: string;
+  expectedTypeWork: ExpectedWorkType;
+  targetWorkCity: string;
+  expectedContractType: ExpectedContractType;
+  expectedSalary: number;
+  canTakeApprenticeship: boolean;
+  monthsOfCommercialExp: number;
+}
+
+export interface StudentCvResponse {
+  firstName: string;
+  lastName: string;
+  bio?: string;
+  githubUsername: string;
+  courseCompletion: string;
+  courseEngagement: string;
+  projectDegree: string;
+  teamProjectDegree: string;
+  portfolioUrls?: string[];
+  projectUrls?: string[];
+  bonusProjectUrls: string[];
+  expectedTypeWork: ExpectedWorkType;
```

Comment:
a mogło by na front iść pole string, a nie ExpectedWorkType, łatwiej będzie korzystać ze wspólnych typów

Reply:
W tym miejscu w sumie warto zadać pytanie czy robimy robimy react-app-rewired dla wspólnych typów ale jak już tyle pracy jest włożone to szkoda babrać się w zmiany wszędzie źródła typów. Widzę na FE folder z typami, więc najłatwiej będzie dodać takiego enuma też tam:

export enum ExpectedWorkType {
  Static = 'Na miejscu',
  CanMoveOut = 'Gotowość do przeprowadzki',
  RemoteOnly = 'Wyłącznie zdalnie',
  Hybrid = 'Hybrydowo',
  Any = 'Bez znaczenia',
}

bo nie widzę sensownej możliwości zmiany ale nie jestem autorem kodu.